packages/eas-cli/src/credentials/ios/actions/SetupMultitargetBuildCredentials.ts

```diff
@@ -0,0 +1,94 @@
+import { IosDistributionType, IosEnterpriseProvisioning } from '@expo/eas-json';
+import chalk from 'chalk';
+import nullthrows from 'nullthrows';
+
+import Log from '../../../log';
+import { CredentialsManager } from '../../CredentialsManager';
+import { Context } from '../../context';
+import { IosCapabilitiesOptions } from '../appstore/ensureAppExists';
+import { App, IosAppBuildCredentialsMap, IosCredentials, Target } from '../types';
+import { displayProjectCredentials } from '../utils/printCredentials';
+import { SetupBuildCredentials } from './SetupBuildCredentials';
+
+interface Options {
+  app: App;
+  targets: Target[];
+  distribution: IosDistributionType;
+  enterpriseProvisioning?: IosEnterpriseProvisioning;
+  iosCapabilitiesOptions?: IosCapabilitiesOptions;
+}
+
+export class SetupMultitargetBuildCredentials {
+  constructor(private options: Options) {}
+
+  async runAsync(ctx: Context): Promise<IosCredentials> {
+    const manager = new CredentialsManager(ctx);
+
+    const hasManyTargets = this.options.targets.length > 1;
+    const iosAppBuildCredentialsMap: IosAppBuildCredentialsMap = {};
+    if (hasManyTargets) {
+      Log.newLine();
```

Comment:
@brentvatne can you review the following logs? 

This is how it looks like now: https://user-images.githubusercontent.com/5256730/119153538-a21f2700-ba51-11eb-9e05-fb349bcd131d.png